meta-luneos/recipes-upstreamable/powerstat/powerstat_git.bb

```diff
@@ -18,3 +18,5 @@ do_compile() {
 do_install() {
     oe_runmake DESTDIR=${D} install
 }
+
+FILES:${PN} += "${datadir}/bash-completion/"
```

Comment:
inherit bash-completion would be better, I had it in my branch, must have pushed older version of this change.

Reply:
Sure, I'll do that